pom.xml

```diff
@@ -27,13 +27,13 @@
   <parent>
     <groupId>org.apache.commons</groupId>
     <artifactId>commons-parent</artifactId>
-    <version>49</version>
+    <version>50</version>
   </parent>
 
   <groupId>org.apache.bcel</groupId>
   <artifactId>bcel</artifactId>
   <packaging>jar</packaging>
-  <version>6.4.1</version>
+  <version>6.4.1.1</version>

```

Comment:
This is not a change from upstream, so it should not be in this pull request.  It should be in a separate commit in typetools/commons-bcel, when we are ready to make a release.